src/script/api/Doxyfile_AI.in

```diff
@@ -10,8 +10,8 @@
 #---------------------------------------------------------------------------
 DOXYFILE_ENCODING      = UTF-8
 PROJECT_NAME           = "OpenTTD AI API"
-PROJECT_NUMBER         = $(VERSION)
-OUTPUT_DIRECTORY       = ../../../docs/aidocs/
+PROJECT_NUMBER         = ${REV_VERSION}
+OUTPUT_DIRECTORY       = ${CPACK_BINARY_DIR}/docs/aidocs/
```

Comment:
`CPACK_BINARY_DIR`? Not `CMAKE_BINARY_DIR`?

Reply:
`CMAKE_BINARY_DIR` is not known inside `FindVersion.cmake` so I used the available variables passed by the target command